### DISCUSSION TITLE
Integrating Sendgrid to Kong Enterprise vis kong.conf

### DISCUSSION THREAD

**Discussion by teraflops-droid** [Kong/kong#9669] [Help] [answered]
I'm having problem setting up smtp server in kong enterprise. After i've put everything that i thought is necessary in smtp config from kong.conf but nothing or email is being delivery
Here is the config i put in 
smtp_mock=off
smtp_host=https://smtp.sendgrid.net/
smtp_port=587
smtp_username=${KONG_SMTP_USER}
smtp_password=${KONG_SMTP_PASSWORD}
smtp_ssl=on
smtp_admin_emails=${password}
admin_emails_from =${password}
portal_invite_email=${password}
portal_access_request_email=${password}
portal_approved_email=on
portal_emails_from=${password}
portal_emails_reply_to=${password}

**Comment by hbagdi** [comment, marked as answer]
Please reach out to our support team at [support.konghq.com](https://support.konghq.com/support/s/) for all Kong Enterprise issues. Thanks!
